index.bs

```diff
@@ -412,10 +412,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
     terminate this algorithm. Otherwise,
     - If the {{ScopedCredentialOptions/rpId}} member of {{options}} is not <a>present</a>, then set |rpId| to |callerOrigin|, 
         and |rpIdHash| to the SHA-256 hash of the UTF-8 encoding of |rpId|.
-    - If the {{ScopedCredentialOptions/rpId}} member of {{options}} is <a>present</a>, then invoke the [[#algo-relax-same-origin-restriction]],
-        using {{ScopedCredentialOptions/rpId}}. If no errors are thrown, set |rpId| to the value of `host` as
+    - If the {{ScopedCredentialOptions/rpId}} member of {{options}} is <a>present</a>, then invoke the [[#algo-relax-same-origin-restriction]]
+        using these arugments: let |relaxingRpId| be {{ScopedCredentialOptions/rpId}}, and let |currentDocument| be the current <a>Document</a>. If no errors are thrown, set |rpId| to the value of `host` as
```

Comment:
is this the style the html et al spec uses to invoke an alg with multiple args?  If not, I'd just copy their extant style (sorry, don't have time now to investigate myself)